content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -222,6 +222,25 @@ This is an informational extension point. Post-bind plugins are called after a
 Pod is successfully bound. This is the end of a binding cycle, and can be used
 to clean up associated resources.
 
+### EnqueueExtension
+
+EnqueueExtension is the interface where the plugin can have control how the scheduling of Pods rejected by this plugin are retried.
+PreEnqueue, PreFilter, Filter, Reserve and Permit plugins are supposed to implement this interface.
+
+{{< feature-state for_k8s_version="v1.28" state="beta" >}}
+
+QueueingHint is a part of EnqueueExtension. 
+It's callback functions to decide if a Pod can be requeued to activeQ/backoffQ. 
```

Comment:
```suggestion
It contains callback functions for deciding whether a Pod can be requeued to the active queue or backoff queue. 
```

Reply:
Looks good except QueueingHint itself is a callback function -- if you say QueueingHint contains callback function, it's kind of weird.